docs/source/reference/core/function_node.rst

```diff
@@ -0,0 +1,8 @@
+FunctionNode
+------------
+
+.. autosummary::
+   :toctree: generated/
+   :nosignatures:
+
+   chainer.FunctionNode
```

Comment:
I feel it is better to move FunctionNode to function.rst. How do you think?

Reply:
@unnonouno Ah, right. I moved it to function.rst and removed function_node.rst.